events/test_routes.py

```diff
@@ -9,7 +9,7 @@
 from app import app, os, connect_to_mongodb
 
 EXAMPLE_TIME = datetime.datetime(
-    2019, 6, 11, 10, 33, 1, 100000).isoformat(" ", "seconds")
+    2019, 6, 11, 10, 33, 1, 100000).isoformat(sep=" ", timespec="seconds")
```

Comment:
You might want to make this `EXAMPLE_TIME_STRING` since it's a string so it matches the global var naming in test_events_class.py.

Reply:
Done for all test files in df59c2b